build.sbt

```diff
@@ -67,9 +69,16 @@ lazy val frontEnd = project("front-end")
       "org.webjars" % "react" % "0.14.3",
       "org.webjars" % "react-router" % "1.0.3",
       "org.webjars" % "jquery" % "2.2.0",
-      "org.webjars" % "foundation" % "5.3.0"
+      "org.webjars" % "foundation" % "5.3.0",
+      "com.typesafe.conductr" %% "lagom10-conductr-bundle-lib" % "1.4.1"
```

Comment:
It is needed because the `lagom10-conductr-bundle-lib` is only added for a `LagomJava` project but not for a `LagomPlay` project. Therefore it needs to be added manually. Soon we will also add support for `LagomPlay` and any other Play project in this regard.